package.json

```diff
@@ -24,33 +22,17 @@
   "author": "",
   "license": "Apache-2.0",
   "dependencies": {
-    "@types/color": "^3.0.0",
-    "@types/extract-zip": "^1.6.2",
-    "@types/inquirer": "^6.5.0",
-    "@types/jasmine": "^3.5.0",
-    "@types/lodash.template": "^4.4.6",
-    "@types/minimist": "^1.2.0",
-    "@types/node": "^12.12.14",
-    "@types/node-fetch": "^2.5.4",
-    "@types/tar": "^4.0.3",
-    "@types/valid-url": "^1.0.2",
-    "color": "^3.1.2",
-    "extract-zip": "^1.6.7",
-    "inquirer": "^7.0.4",
-    "jimp": "^0.9.3",
-    "lodash.template": "^4.5.0",
-    "minimist": "^1.2.0",
-    "node-fetch": "^2.6.0",
-    "npm": "^6.13.7",
-    "tar": "^5.0.5",
-    "typescript": "^3.7.2",
-    "valid-url": "^1.0.9"
+    "@llama-pack/core": "file:packages/core",
+    "@llama-pack/cli": "file:packages/cli"
   },
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "^2.9.0",
     "@typescript-eslint/parser": "^2.9.0",
     "eslint": "^6.6.0",
     "eslint-config-google": "^0.14.0",
-    "jasmine": "^3.5.0"
+    "jasmine": "^3.5.0",
```

Comment:
So the tests are in the parent package?

Reply:
We don't cleaned-up the dependencies in root. We can re-add later if we end up needing.